run_all_tests.py

```diff
@@ -1,3 +1,3 @@
 import os
 
-os.system('cmd /c "python -m pytest"')
\ No newline at end of file
+os.system('cmd /c "python -m pytest -s"')
```

Comment:
What does this -s do?

Reply:
Removes the capturing of the print() statements, to allow logging